.project

```diff
@@ -10,9 +10,15 @@
 			<arguments>
```

Comment:
I guess this file could probably be .gitignore'd (I thought it was already in github's default .gitignore for Dart), not that it hurts anything having it.